### PR TITLE
fix: Give warnings when deprecated template tag include_static is used

### PR DESCRIPTION
**Pull request by abhiabhi94**

### CODE REVIEW

File: comment/templatetags/comment_tags.py [hunk 1]
```diff
@@ -139,9 +140,17 @@ def is_user_blocked(comment):
...
 
 
+# TODO: remove in v3.0.0
 @register.simple_tag(name='include_static')
 def include_static():
     """ This function shall be deprecated """
+    warnings.warn(
+        (
+            'The tag `include_static` has been deprecated. Static files are now rendered implicitly.'
+            'You can remove this from your django template. This tag will be removed in a v3.0.0.'
```
Comment (Radi85) [inline comment/templatetags/comment_tags.py:150]: Can you open an issue for this change and assign _3.0.0_ milestone?

**Review by Radi85 — CHANGES_REQUESTED** [review]

**Review by Radi85 — DISMISSED** [review]

File: comment/templatetags/comment_tags.py (hunk 1 above)
Comment (abhiabhi94) [reply comment/templatetags/comment_tags.py:150]: #216 aims to track this.

**Review by Radi85 — DISMISSED** [review]

**Comment by Radi85** [comment]
@abhiabhi94,
Can you please rebase this PR?

**Comment by Radi85** [comment]
Hey @abhiabhi94, I forgot to mention this before, it would be nice if we open an issue for any change/new commit for better documentation purpose.

**Comment by abhiabhi94** [comment]
> Hey @abhiabhi94, I forgot to mention this before, it would be nice if we open an issue for any change/new commit for better documentation purpose.

I will try to keep this in mind for future purposes.

**Review by Radi85 — APPROVED** [review]

**Comment by abhiabhi94** [comment]
Thanks, Radico, for checking.

**Pull request merged**